### ISSUE TITLE
Build 'googlecompute' errored: unexpected EOF

### ISSUE THREAD

**Issue by JorritSalverda** [hashicorp/packer#3378]
When running multiple packer jobs from the same host simultaneously it frequently fails with an unexpected EOF.

This can happen at different moments during the packer build. Often when uploading a file to the machine, but it also happens when wrapping up and saving the image as can be seen in the linked gist.
- Packer Version: 0.10.0
- Host platform: debian jessie
- Debug log output : https://gist.github.com/JorritSalverda/5f0fa8e942bf0b844f7c

**Comment by JorritSalverda** [comment]
I don't recall this happening in version 0.9.0, so might be a newly introduced issue.

**Comment by JorritSalverda** [comment]
Another gist for when the error happens during file upload:

https://gist.github.com/JorritSalverda/1f280981b1933881e8cc

**Comment by JorritSalverda** [comment]
Going back to 0.9.0 - and using pd-standard instead of pd-ssd - didn't help, so investigating other changes I did recently.

**Comment by JorritSalverda** [comment]
Reducing the amount of parallel packer builds to run from the same host to 4 instead of the 5 I ran before seems to have stabilized the bake process. The host might have run out of some type of resource, although I'm not sure which one.

**Comment by AlexFinn** [comment]
I have the same problem:
Packer Version: 0.10.1
Target Host: ubuntu 14.04

**Comment by AlexFinn** [comment]
with PACKER_LOG=1
```
2016/07/12 13:20:52 [ERR] yamux: keepalive failed: i/o deadline reached
2016/07/12 13:20:52 ui error: Build 'gcloud' errored: unexpected EOF
2016/07/12 13:20:52 Builds completed. Waiting on interrupt barrier...
2016/07/12 13:20:52 machine readable: error-count []string{"1"}
2016/07/12 13:20:52 ui error:
==> Some builds didn't complete successfully and had errors:
2016/07/12 13:20:52 machine readable: gcloud,error []string{"unexpected EOF"}
2016/07/12 13:20:52 [INFO] 3614 bytes written for 'stdout'
2016/07/12 13:20:52 ui error: --> gcloud: unexpected EOF
2016/07/12 13:20:52 [ERR] Error decoding response stream 21: EOF
2016/07/12 13:20:52 ui:
==> Builds finished but no artifacts were created.
2016/07/12 13:20:52 waiting for all plugin processes to complete...
2016/07/12 13:20:52 [INFO] RPC endpoint: Communicator ended with: 123
2016/07/12 13:20:52 Error in Ui RPC call: unexpected EOF
2016/07/12 13:20:52 [INFO] 0 bytes written for 'stderr'
Build 'gcloud' errored: unexpected EOF

==> Some builds didn't complete successfully and had errors:
--> gcloud: unexpected EOF

==> Builds finished but no artifacts were created.
2016/07/12 13:20:53 packer: 2016/07/12 13:20:52 [INFO] 3614 bytes written for 'stdout'
2016/07/12 13:20:53 packer: 2016/07/12 13:20:52 [INFO] RPC client: Communicator ended with: 123
2016/07/12 13:20:53 packer: 2016/07/12 13:20:52 [INFO] 0 bytes written for 'stderr'
```

**Comment by mDantas** [comment]
I'm seeing the same issue, also without multiple packer jobs. Anecdotally, it feels like this happens ~25% of the time.

Packer 1.2.3
Target Host: ubuntu 16.04

**Comment by SwampDragons** [comment]
My guess is that this is a duplicate of the linked issue  #5360, so I am going to close this.
